compiler/fir/raw-fir/light-tree2fir/src/org/jetbrains/kotlin/fir/lightTree/converter/DeclarationsConverter.kt

```diff
@@ -1017,16 +1017,15 @@ class DeclarationsConverter(
                                 it.containingClassAttr = lookupTag
                             }
                         }
-                    this.setter =
-                        if (isVar) {
-                            convertedAccessors.find { it.isSetter }
-                                ?: FirDefaultPropertySetter(
-                                    null, session, FirDeclarationOrigin.Source, returnType, propertyVisibility
-                                ).also {
-                                    currentDispatchReceiverType()?.lookupTag?.let { lookupTag ->
-                                        it.containingClassAttr = lookupTag
-                                    }
+                    this.setter = convertedAccessors.find { it.isSetter }
```

Comment:
Minor, but isn't it still worth leaving some comments here (and corresponding part in `RawFirBuilder`) that we still need to build a setter for `val` to report `VAL_WITH_SETTER`?